sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -1,15 +1,16 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (2022-02-10)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+- Added interfaces from `com.azure.core.client.traits` to `ConfigurationClientBuilder`.
```

Comment:
We should call out that this introduced a new `retryOptions` method in the builder.